python-package/xgboost/spark/core.py

```diff
@@ -597,6 +597,9 @@ def _get_unwrapped_vec_cols(feature_col: Column) -> List[Column]:
 )
 
 
+_MODEL_CHUNK_SIZE = 4096 * 1024
```

Comment:
I was wondering if 4M is an appropriate size for the normal size model? @trivialfis any idea about it?

Reply:
the chunk size is for making spark processing easier, i.e. one chunk becomes one row in spark dataframe. So the chunk_size should be determined by the proper spark row size right ?

Reply:
Yeah, make sense, so what's the proper spark row size that can make spark process easier?

If the model size is like 1G, then it will be split to 256 rows. should that have some perf issue?

Reply:
no perf issue. very safe. it just optimizes the row size so that it improves pyspark dataframe ser / deser processing.

Reply:
4MB is a good row size according to our experience